### PR TITLE
Syndicate bundle balance [DNM]

### PR DESCRIPTION
**Pull request by harmonyn**
BUCKLE UP KIDS, THERE'S GONNA BE SOME CONTROVERSY IN THESE HERE WATERS

All of this PR is tentative. All of it is subject to change.

tl;dr: Syndicate bundles have been very neglected in light of new items, TC costs and such. I have made a few changes. Here are some of them.

NEW BUNDLE:

Sabotage! Designed for traitors to sabotage the station. Tentative contents:
- A powersink
- Two C4s
- Two door charges
- A camera bug
- A Detomax

NEW WEIGHTS (compared to before, where everything had the same chance):

3: 
- Fake Nuke Ops
- Spy
- Stealth
- Sabotage

2: 
- Implant
- Hacker
- Singularity
- Murder

1:
- Bond
- Guns
- Sith Lord (the worst of the lot, literally a DOUBLE ESWORD MURDERBONER LASTING 10 MINUTES TIME bundle, I want to remove it entirely but preliminary feedback was negative)

NEW ITEMS:
- AI detector and Syndicate encryption key added to Spy bundle to make it suck less. Removed one agent card because there were two.
- Radioactive laser and soap added to Stealth bundle, which took a lot of nerfs with ebow AND sleepypen being nerfed.
- Syndicate encryption key added to Fake Nuke Ops bundle.
- AI detector added to Hacker bundle.

ADD YOUR THOUGHTS BELOW OR DON'T BITCH WHEN THEY GO THROUGH.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
honestly i would be pissed if i got the sabotage bundle
oh no, you can blow up two doors and a few walls! you can blow up your own PDA with the rng-based detomemetrix! you can make the station go without power for a few minutes, the horror!
at least throw in like a minibomb or something

**Comment by Lumipharon** [comment]
One of the bundles has TWO FUCKING AGENT CARDS.
I think it's the spy bundle. It is also utterly redundant and every time I get it, I die a little inside.

**Comment by bgobandit** [comment]
>Removed one agent card because there were two.

**Comment by Dorsisdwarf** [comment]
The two agent cards is so you can swap out your fake identity once ":s beep boop jimmy traitorkins killed a guy and ran into arrivals maint"

**Comment by SconesC** [comment]
Keep the weights at 1?

**Comment by Lumipharon** [comment]
You can change what's on your agent card though.

File: code/game/machinery/syndicatebeacon.dm [hunk 1]
```diff
@@ -218,3 +218,7 @@
 /obj/item/device/sbeacondrop/bomb
 	desc = "A label on it reads: <i>Warning: Activating this device will send a high-ordinance explosive to your location</i>."
 	droptype = /obj/machinery/syndicatebomb
+
+/obj/item/device/sbeacondrop/powersink
+	desc = "A label on it reads: <i>Warning: Activating this device will send a high-ordinance explosive to your location</i>."
```
Comment (Core0verload) [inline code/game/machinery/syndicatebeacon.dm]: Change the desc.

**Comment by Durkel** [comment]
I think you should replace the camera bug with a minibomb in the sabotage bundle, just my two cents.

**Comment by Zilenan91** [comment]
Detomatix RnG should just be removed imo. There's nothing worse than it happening on the first try and wasting all your TCs

**Comment by Xhuis** [comment]
You should replace the Syndicate key in the fake ops bundle with a nuke op headset.

**Comment by Razharas** [comment]
@bgobandit 
The status of this?

**Comment by Razharas** [comment]
@bgobandit

**Comment by Razharas** [comment]
@bgobandit cmon

**Comment by Razharas** [comment]
Whatevs fams

**Comment by Razharas** [comment]
Whatevs fams
